app.py

```diff
@@ -602,6 +600,21 @@ def narrative_status():
     return(flask.Response(json.dumps(resp_doc), 200, mimetype='application/json'))
 
 
+@app.route("/reaper/", methods=['GET'])
```

Comment:
One way we could secure this endpoint is with the admin role.  Then whatever requested the /reaper/ URL would need an appropriate KBase token.  I am honestly not sure what mode is better, using a token or hiding the container running the reaper.

Reply:
...actually if we're having Rancher call /reaper.../ as a health check it may not be able to provide a token.  I don't see any options for setting a header in the UI.

Alternatively we could have a check_mk HTTP rule to call /reaper..../.  check_mk does support setting headers.